collection-scripts/gather_nodes

```diff
@@ -33,7 +33,7 @@ do
     NODE_PATH=${NODES_PATH}/$node/
     mkdir -p ${NODE_PATH}
     oc exec -it $pod -n node-gather -- ip a >> $NODE_PATH/ip.txt
-    oc exec -it $pod -n node-gather -- brctl show >> $NODE_PATH/bridge
+    oc exec -it $pod -n node-gather -- bridge link >> $NODE_PATH/bridge
```

Comment:
Can we add bridge vlan show here ?

Reply:
As far as I understand the collection scripts were complete. I do not see information provided by `bridge vlan show` in `brctl show`.  It is a replacement to get rid of bridge-utils dependency. Do you want me to add it?

Reply:
Comparing the result of `btctl show` vs `bridge link` shows the port-less bridges aren't reported by `bridge link`:
```
masayag@masayag: ~$ brctl show
bridge name     bridge id               STP enabled     interfaces
docker0         8000.0242ebcc5033       no
foreman         8000.54540047130b       yes             foreman-nic
tempbr          8000.000000000000       no
virbr0          8000.525400029353       yes             virbr0-nic
virbr3          8000.525400ab8f2e       yes             virbr3-nic
```
vs
```
masayag@masayag: ~$ bridge link
5: foreman-nic: <BROADCAST,MULTICAST> mtu 1500 master foreman state disabled priority 32 cost 100 
7: virbr0-nic: <BROADCAST,MULTICAST> mtu 1500 master virbr0 state disabled priority 32 cost 100 
9: virbr3-nic: <BROADCAST,MULTICAST> mtu 1500 master virbr3 state disabled priority 32 cost 100
```
Please add the following to record all of the bridges (w/o ports):
`oc exec -it $pod -n node-gather -- ip -o  link show type bridge >> $NODE_PATH/bridge`

The output is:
```
4: foreman: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN mode DEFAULT group default qlen 1000\    link/ether 54:54:00:47:13:0b brd ff:ff:ff:ff:ff:ff
6: virbr0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN mode DEFAULT group default qlen 1000\    link/ether 52:54:00:02:93:53 brd ff:ff:ff:ff:ff:ff
8: virbr3: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN mode DEFAULT group default qlen 1000\    link/ether 52:54:00:ab:8f:2e brd ff:ff:ff:ff:ff:ff
10: docker0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN mode DEFAULT group default \    link/ether 02:42:eb:cc:50:33 brd ff:ff:ff:ff:ff:ff
11: tempbr: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN mode DEFAULT group default qlen 1000\    link/ether 5a:56:56:1e:99:50 brd ff:ff:ff:ff:ff:ff
```

Reply:
Make sense, will update

Reply:
@pkliczewski about bridge vlan show I know it was not with brctl but now that we are using iproute would be nice to add this information since we are going to configure vlan_filtering at linux bridges we will need this for debugging purposes.

You can dump it as json bridge -j vlan show

Reply:
Since this PR is only about removing stale dependency. Are you OK to push `bridge -j vlan show` in next PR?

Reply:
ack will do thanks!